workspaces/adventure-pack/goodies/typescript/Math.gcd/index.test.ts

```diff
@@ -0,0 +1,19 @@
+import { describe, expect, it } from "@jest/globals";
+
+import "./index";
+
+describe("Math.gcd", () => {
+  it.each([
```

Comment:
Could we add a couple of test cases using much larger numbers? A test case using `10 ** 9 + 7` would be nice since that number is used so often in programming problems. (Spoiler alert, it's a prime.)